### PR TITLE
AP_HAL_ChibiOS: update default params for SWAN-K1

### PR DESCRIPTION
**Pull request by Hwurzburg**
Some modifications:
- change to new copter surfaceless tailsitter enable param
- adjust rudder extremes to allow stick arm/disarm in T10 transmitter...it limits rudder a bit at lowest throttle
- create throttle trim to accommodate #18516 when merged...no change to behavior if not merged
- put full tune params for reset to default to get back to base tune
- backtransition rate lower to stop time out
- change QRTL option to be button B on both transmitters
- add TRIM_THROTTLE target for #18516

### CODE REVIEW

File: libraries/AP_HAL_ChibiOS/hwdef/Swan-K1/defaults.parm [hunk 1]
```diff
@@ -110,16 +115,20 @@ Q_A_THR_MIX_MIN  0.3
 Q_RTL_MODE 3
 Q_TRANS_DECEL 2.5
 
-# QRTL on 'D' button
+# QRTL on 'D' button on H12
 RC10_OPTION 4
 
+# QRTL on "B" button on T10
+RC8 4
```
Comment (tridge) [inline libraries/AP_HAL_ChibiOS/hwdef/Swan-K1/defaults.parm:122]: typo, RC8_OPTION

**Review by tridge — CHANGES_REQUESTED** [review]

File: libraries/AP_HAL_ChibiOS/hwdef/Swan-K1/defaults.parm (hunk 1 above)
Comment (Hwurzburg) [reply libraries/AP_HAL_ChibiOS/hwdef/Swan-K1/defaults.parm]: fixed per our discussion

**Comment by IamPete1** [comment]
All seems fine, since it has been flowing IRL. I'm slightly hesitant about having RC trim/min/max in the defaults I'm not sure how similar they would end up over different vehicles, possibly there are different transmitter options.

**Comment by Hwurzburg** [comment]
> All seems fine, since it has been flowing IRL. I'm slightly hesitant about having RC trim/min/max in the defaults I'm not sure how similar they would end up over different vehicles, possibly there are different transmitter options.

this is a vehicle hwdef...it comes with transmitters....their trims and ranges are known....you would only use this with the Swan K1 RTF setups

**Comment by Hwurzburg** [comment]
per our conv...going back to original Single/Dual

**Review by tridge — APPROVED** [review]

**Pull request merged**